src/utils/data/animals.mock.json

```diff
@@ -156,6 +157,17 @@
       "tags": ["small"],
       "type": "parameters"
     },
+    {
```

Comment:
Given that this dataset is also used with the dataset for the kedro-viz server, adding new fields to this file might cause the python tests to fail and will likely need updates to their test files. You might need to liase with the core team / whoever is maintaining the server codebase (cc @limdauto ) to check on this.

Reply:
Please feel free to modify these files to test the frontend. We don't rely on them for the backend anymore.